grakn-graph/src/test/java/ai/grakn/generator/GraknGraphs.java

```diff
@@ -266,29 +246,20 @@ public GraknGraphs setOpen(boolean open) {
             },
             () -> {
                 Relation relation = relation();
-                Instance instance = instance();
-                relation.scope(instance);
-                summary(relation, "scope", instance);
-            },
```

Comment:
I don't think this merge is correct. It looks like we're no longer generating graphs with scope edges. I'll fix this one.